vcam-util.c

```diff
@@ -189,7 +193,7 @@ int main(int argc, char *argv[])
             break;
         case 's':
             if (!parse_resolution(optarg, &dev)) {
-                printf("Failed to parse resolution");
+                printf("Failed to parse resolution\n");
```

Comment:
Later, you can send another patch which unifies the output of error messages, from stdout to stderr.

Reply:
Ok I'll send this messages issue on another patch.